elections-api/src/main/java/me/lorenzo0111/elections/api/objects/ElectionBlock.java

```diff
@@ -1,7 +1,7 @@
 /*
  * This file is part of ElectionsPlus, licensed under the MIT License.
  *
- * Copyright (c) Lorenzo0111
+ * Copyright (©) tadhunt
```

Comment:
Can you fix the copyright?

Reply:
Sure!

What do you want here?
* Change the © to (c)
* Change tadhunt to Lorenzo0111?
* Both things?

Since I authored this file and there isn't a contributors policy, I wasn't sure what the right thing to do is.

Reply:
I assumed I should fix both, so those changes are now also in this PR (3dce35260356e0f597e4315daeaf072359131d41)